src/components/info-tile/info-tile.tsx

```diff
@@ -113,6 +119,7 @@ export class InfoTile {
                 tabindex="0"
                 aria-label={extendedAriaLabel}
                 title={this.linkTitle}
+                class={{ 'clickable-infotile': !!this.href || this.clickable }}
```

Comment:
Good idea! However, I think this should be more automated, like if there is an onClick defined. Maybe that's not possible. Given the fact that that can be set on the component from outside as well 🤔 We can wait to see what we will end up doing for `limel-dashabord-shortcut`

Also, if we have this, we can also check for `!this.disabled` in the same place, and remove that `box-shadow: unset` rule from the CSS

Reply:
for these kind of classes, we use a convention like: `is-xxx` or `has-xxx`, so `is-clickable` would be a better class name

Reply:
> Good idea! However, I think this should be more automated, like if there is an onClick defined. Maybe that's not possible. Given the fact that that can be set on the component from outside as well 🤔 We can wait to see what we will end up doing for `limel-dashabord-shortcut`

I see your point, I also started from the `onClick`, but as you said it's not quite possible to manage at the Elements side so I talked to @adrianschmidt, and his suggestion is to set up an extra prop, so we can apply the style based on it.

Reply:
> Also, if we have this, we can also check for `!this.disabled` in the same place, and remove that `box-shadow: unset` rule from the CSS

Agree! will add a fixup in a sec